.appveyor.yml

```diff
@@ -28,7 +28,7 @@ cache:
   - '%R_LIBS_USER%'
 
 build_script:
-  - set PATH=C:\Program Files\Git\mingw64\bin;%PATH%
+  - set PATH=C:\Program Files\Git\mingw64\bin;%R_HOME%\bin;%PATH%
```

Comment:
This is OK if there's no other viable alternative, but this is a "hack" and we should probably even have it on a separate line and mark it as a (temporary) hack via a comment.